samples/drivers/jesd216/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&uart136 {
+	status = "okay";
+};
```

Comment:
why is this needed?

Reply:
I think it is a leftover. I see that `uart136` is enabled by default now.

Reply:
Removed